test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+yarn run test & \
```

Comment:
the `\` and next indentation isn't needed here, the next lines are their own command